zk_toolbox/crates/zk_supervisor/src/commands/test/revert.rs

```diff
@@ -15,7 +15,11 @@ pub fn run(shell: &Shell, args: RevertArgs) -> anyhow::Result<()> {
     shell.change_dir(ecosystem_config.link_to_code.join(REVERT_TESTS_PATH));
 
     logger::info(MSG_REVERT_TEST_RUN_INFO);
-    install_and_build_dependencies(shell, &ecosystem_config)?;
+
+    if args.no_deps {
+        install_and_build_dependencies(shell, &ecosystem_config)?;
```

Comment:
install if no deps?

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2594/commits/1d8282816a3aa52dbcbc7d68c12d4f0775288b41